payonline.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.12'
+  spec.add_development_dependency 'activesupport', '~> 4.2.6'
```

Comment:
I have project with version activesupport = 4.1.
I think the best code is 
spec.add_development_dependency 'activesupport', '>= 3.2.0'